learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md

```diff
@@ -102,25 +102,30 @@ Follow the steps below to start a remote debug session.
 
 2. Open the Terminal and execute the Ballerina command, which you want to debug, out of the supported remote debugging commands below. 
 
-    - Debugging a Ballerina package/ single file: 
+    - Debugging a Ballerina package or a single file: 
 
     ```bash
     bal run --debug <DEBUGGEE_PORT> <BAL_FILE_PATH/PACKAGE_PATH>
     ```
 
-    - Debugging ballerina tests: 
+   - Debugging Ballerina executable jar:  
+
+    ```bash 
+    bal run --debug <DEBUGGEE_PORT> <JAR_FILE_PATH>
```

Comment:
```suggestion
    bal run --debug <DEBUGGEE_PORT> <EXECUTABLE_PATH>
```

Reply:
https://github.com/ballerina-platform/ballerina-dev-website/pull/2189#discussion_r636828843

Reply:
Shall we say `<EXECUTABLE_JAR_FILE_PATH>`?

Reply:
@praneesha +1

Reply:
Updated in f8d2254